src/vscode-bicep-ui/packages/components/src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,11 @@
+import type { PropsWithChildren } from "react";
+import type { AccordionItemId } from "./types";
+
+import { useState } from "react";
+import { AccordionProvider } from "./AccordionProvider";
+
+export function Accordion({ children }: PropsWithChildren) {
```

Comment:
Out of scope for now, but do you think it's worth trying to (or makes sense to) contribute this to https://github.com/microsoft/vscode-webview-ui-toolkit in the long run?

Reply:
It may be challenging to contribute since `vscode-webview-ui-toolkit` is built on top of a native Web Components library [microsoft/fast](https://github.com/microsoft/fast), which essentially means Vanilla TS/JS. For React they provide a wrapper [@microsoft/fast-react-wrapper](https://www.npmjs.com/package/@microsoft/fast-react-wrapper) to generate React bindings. To contribute, we would need to learn Web Components, become familiar with the fast library, and then rewrite the entire component.